yeezyCopper.js

```diff
@@ -60,69 +64,153 @@ function pay () {
     }
   });
 
-  $('input#dwfrm_payment_creditCard_cvn').val(function () {
+  $('input#dwfrm_payment_creditCard_cvn').val(function() {
     return config.security_code;
   });
 
-  setTimeout(function () {
+  setTimeout(function() {
     placeOrder();
   }, 300);
 }
+
+function _x(STR_XPATH) {
+  var xresult = document.evaluate(STR_XPATH, document, null, XPathResult.ANY_TYPE, null);
+  var xnodes = [];
+  var xres;
+  while (xres = xresult.iterateNext()) {
+    xnodes.push(xres);
+  }
+
+  return xnodes;
 }
 
-function addNewShipping() {
-  $('.textinput.firstname').val(function () {
-    return config.first_name;
-  });
+function sleep(milliseconds) {
+  var start = new Date().getTime();
+  for (var i = 0; i < 1e7; i++) {
+    if ((new Date().getTime() - start) > milliseconds) {
+      break;
+    }
+  }
+}
 
-  $('.textinput.lastname').val(function () {
-    return config.last_name;
-  });
 
-  $('.textinput.address1').val(function () {
-    return config.street_address_1;
-  });
+function checkovernightShip() 
+{
+  //first need to check shipping speed, for overnight shipping or normal
+  console.log('First Checking Shipping Speed');
+  if (shipfast === true) {
```

Comment:
so unless youre strictly checking that shipfast is set to `true` and not just not undefined, null, 0 you can write this as `if (shipfast) { ... }`